sdks/python/apache_beam/ml/rag/ingestion/mysql_common.py

```diff
@@ -134,6 +134,11 @@ def json(
     return cls(column_name, str, value_fn)
 
 
+def embedding_to_string(embedding: List[Float]) -> str:
```

Comment:
```suggestion
def embedding_to_string(embedding: List[float]) -> str:
```

I think this is causing failures